### ISSUE TITLE
--version

### ISSUE THREAD

**Issue by sublee** [sharkdp/fd#45]
Please provide `--version` option to print the version.

```console
$ fd --version
1.1.0
```

**Comment by sharkdp** [comment]
This was already supported on master. I've just released [v2.0.0](https://github.com/sharkdp/fd/releases/tag/v2.0.0), which includes this:
``` purs
> fd -V      
fd 2.0.0

> fd --version
fd 2.0.0
```

**Comment by sublee** [comment]
Oh, awesome!  Thank you!
